classes/components/listPanels/PKPEmailTemplatesListPanel.inc.php

```diff
@@ -13,6 +13,7 @@
  */
 namespace PKP\components\listPanels;
 use PKP\components\listPanels;
+define('EMAIL_TEMPLATE_STAGE_DEFAULT', 0);
```

Comment:
Hmm, I'm wondering if this is the right place to put this constant. Since it's used in the REST API I think we should move it to the service class, `PKPEmailTemplateService`.

Reply:
Agree